languages/cpp/examples/Wrapper.cpp

```diff
@@ -24,51 +28,91 @@ int main() {
-    // // Access token login
-    bitwardenClient.loginAccessToken(accessToken);
-    // Organization ID
-    boost::uuids::uuid organizationUuid = boost::uuids::string_generator()(organizationId);    
-    
-    // // Create a new project
+    BitwardenClient bitwardenClient(bitwardenSettings);
+
+    // Access token login
+    bitwardenClient.loginAccessToken(accessToken, stateFile);
+
+    // Convert organization ID to UUID
+    boost::uuids::uuid organizationUuid = boost::uuids::string_generator()(organizationId);
+
+    // Create a new project
+    std::cout << "Projects:\n";
     ProjectResponse projectResponseCreate = bitwardenClient.createProject(organizationUuid, "NewTestProject");
     boost::uuids::uuid projectId = boost::uuids::string_generator()(projectResponseCreate.get_id());
-   
+
+    std::cout << "\tcreateProject: '" << projectResponseCreate.get_name() << "'\n\n";
+
     // List projects
     ProjectsResponse projectResponseList = bitwardenClient.listProjects(organizationUuid);
+    std::cout << "\tlistProjects:\n";
+    for (const ProjectResponse& project : projectResponseList.get_data()) {
+        std::cout << "\t\tID: '" << project.get_id() << "', Name: '" << project.get_name() << "'\n";
+    }
+    std::cout << '\n';
 
     // Get project details
     ProjectResponse projectResponseGet = bitwardenClient.getProject(projectId);
+    std::cout << "\tgetProject:\n\t\tID: '" << projectResponseGet.get_id() << "', Name: '" << projectResponseGet.get_name() << "'\n\n";
 
     // Update project
-    ProjectResponse ProjectResponseUpdate = bitwardenClient.updateProject(projectId, organizationUuid, "NewTestProject2");
+    ProjectResponse projectResponseUpdate = bitwardenClient.updateProject(organizationUuid, projectId, "NewTestProject2");
+    std::cout << "\tupdateProject: '" << projectResponseUpdate.get_name() << "'\n\n";
 
     // Secrets
     std::string key = "key";
     std::string value = "value";
     std::string note = "note";
 
+    // Sync secrets
+    std::cout << "Secrets:\n";
+    std::cout << "\tSyncing secrets...\n";
+    std::chrono::system_clock::time_point lastSyncedDate = std::chrono::system_clock::now();
+    SecretsSyncResponse secretsSyncResponse = bitwardenClient.sync(organizationUuid, lastSyncedDate);
+    std::cout << "\tSync has changes: '" << (secretsSyncResponse.get_has_changes() ? "true" : "false") << "'\n\n";
+
     // Create a new secret
-    SecretResponse secretResponseCreate = bitwardenClient.createSecret(key, value, note, organizationUuid, {projectId});
+    SecretResponse secretResponseCreate = bitwardenClient.createSecret(organizationUuid, key, value, note, {projectId});
     boost::uuids::uuid secretId = boost::uuids::string_generator()(secretResponseCreate.get_id());
 
+    std::cout << "\tcreateSecret: '" << secretResponseCreate.get_key() << "'\n\n";
+
     // List secrets
     SecretIdentifiersResponse secretIdentifiersResponse = bitwardenClient.listSecrets(organizationUuid);
 
```

Comment:
Should we std out results of `secretIdentifiersResponse` ?

Reply:
Addressed in 3a7cd09e9d1d7fcf19ef5c9664552cd552ad9dcb.